src/ucp/core/ucp_mm.c

```diff
@@ -452,36 +466,38 @@ ucs_status_t ucp_mem_type_reg_buffers(ucp_worker_h worker, void *remote_addr,
     ucs_status_t status;
     char *rkey_buffer;
 
-    tl_md = &context->tl_mds[md_index];
-    cmpt  = context->tl_cmpts[tl_md->cmpt_index].cmpt;
+    md_attr = &context->tl_mds[md_index].attr;
```

Comment:
use inline init

Reply:
what do you mean? Set it during initialization?

Reply:
yes